### ISSUE TITLE
Crash on 1.8

### ISSUE THREAD

**Issue by DeathsGun** [WurstAPI/WurstAPI#4]
This caused by the command and mod manager they try to register mods for version 1.11 and 1.12

probably version 1.9 is included in this error

```
---- Wurst Client Crash Report ----
// I bet Cylons wouldn't have this problem.

Time: 21.06.17 18:14
Description: Initializing game

java.lang.NoClassDefFoundError: net/wurstclient/features/mods/BoatFlyMod
	at net.wurstclient.features.mods.ModManager.<init>(ModManager.java:56)
	at net.wurstclient.WurstClient.startClient(WurstClient.java:47)
	at net.minecraft.client.Minecraft.startGame(Minecraft.java:506)
	at net.minecraft.client.Minecraft.run(Minecraft.java:340)
	at net.minecraft.client.main.Main.main(Main.java:127)
Caused by: java.lang.ClassNotFoundException: net.wurstclient.features.mods.BoatFlyMod
	at java.net.URLClassLoader.findClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	... 5 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.wurstclient.features.mods.ModManager.<init>(ModManager.java:56)
	at net.wurstclient.WurstClient.startClient(WurstClient.java:47)
	at net.minecraft.client.Minecraft.startGame(Minecraft.java:506)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.Minecraft.run(Minecraft.java:340)
	at net.minecraft.client.main.Main.main(Main.java:127)

-- System Details --
Details:
	Minecraft Version: 1.8
	Wurst Client Version: 6.7
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_131, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 386906368 bytes (368 MB) / 523501568 bytes (499 MB) up to 17166499840 bytes (16371 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx16G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: Wurst MC 1.8-API
	LWJGL: 2.9.1
	OpenGL: GeForce GTX 1070/PCIe/SSE2 GL version 4.5.0 NVIDIA 382.05, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
```

**Comment by DeathsGun** [comment]
Maybe reflection? We could use reflection scan the net.wurstclient.features.mods package for Mod classes and register them.

**Comment by DeathsGun** [comment]
Fixed in 1.4
